.github/workflows/publish-snapshot.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish snapshot
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish-snapshot:
+    runs-on: ubuntu-latest
+    if: github.repository == 'square/anvil'
+    timeout-minutes: 25
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: gradle/wrapper-validation-action@v2
+      - uses: actions/setup-java@v2
```

Comment:
We had `- publish-maven-local` here before, was that just an optimization to let us reuse the results of the local publishing step or are we losing out on anything but not being able to depend on it now?

Reply:
Even before the change, it wouldn't have given any real benefit.  `ci.yml` doesn't do any caching/restoring of the build artifacts, so the jars and whatnot created by the local publishing task are all just discarded.  That `publish-maven-local` shard is essentially just a way to ensure that the publishing configuration works.

Reply:
It's never a good idea to rely upon a dynamically-versioned dependency like a snapshot.  I would hope that if anyone is using our snapshots like that, they're also using [specifically pinned versions](https://medium.com/@vanniktech/pinning-a-snapshot-dependency-to-a-specific-version-b477f79469fc) to ensure consistency.